packages/e2e-tests-playwright/playwright.config.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import path from 'path';
+import { devices } from '@playwright/test';
+import type { PlaywrightTestConfig } from '@playwright/test';
+
+const config: PlaywrightTestConfig = {
+	reporter: process.env.CI
+		? [ [ 'github' ], [ './config/flaky-tests-reporter.ts' ] ]
+		: 'list',
+	forbidOnly: !! process.env.CI,
+	workers: 1,
+	retries: process.env.CI ? 2 : 0,
+	timeout: parseInt( process.env.TIMEOUT || '', 10 ) || 100_000, // Defaults to 100 seconds.
+	// Don't report slow test "files", as we will be running our tests in serial.
+	reportSlowTests: null,
+	testDir: new URL( './specs', 'file:' + __filename ).pathname,
```

Comment:
The code might seem weird, but it makes it easier to swap into ESM (`import.meta.url`) in the future if needed.